src/places/components/PlaceEdit.vue

```diff
@@ -25,6 +25,55 @@
             </template>
           </QInput>
 
+          <QSelect
+            v-model="edit.placeType"
+            :options="placeTypeOptions.filter(({ status, value }) => status === 'active' || edit.placeType === value)"
+            map-options
+            emit-value
+            :label="$t('STOREEDIT.PLACE_TYPE')"
+            outlined
+            class="q-mb-lg"
+          >
+            <template #before>
+              <QIcon name="fas fa-eye" />
```

Comment:
This is the copy-and-paste eye I guess? ... and my suggestion is actually to remove all the #before icons....